spidermon/contrib/monitors/mixins/validation.py

```diff
@@ -193,16 +204,20 @@ def check_field_errors_percent(self, field_name, errors=None, allowed_percent=0)
             )
         else:
             errors_percent = self.validation.fields[field_name].errors.percent
-        self._check_field_errors_percent(errors_percent, allowed_percent)
-
-    def _check_field_errors_percent(self, errors_percent, allowed_percent):
-        self.assertLessEqual(
-            errors_percent,
-            allowed_percent,
-            msg="{percent}% of fields have validation errors!{threshold_info}".format(
-                percent=errors_percent * 100,
-                threshold_info=(" (maximum allowed %.0f%%)" % (allowed_percent * 100))
-                if allowed_percent > 0
-                else "",
-            ),
-        )
+        self._check_field_errors_percent(errors_percent, field_name, allowed_percent)
+
+    def _check_field_errors_percent(self, errors_percent, field_name, allowed_percent):
+        try:
+            self.assertLessEqual(
+                errors_percent,
+                allowed_percent,
+                msg="{percent}% of field {field_name} have validation errors!{threshold_info}".format(
+                    percent=errors_percent * 100,
+                    field_name=field_name,
+                    threshold_info=(" (maximum allowed %.0f%%)" % (allowed_percent * 100))
+                    if allowed_percent > 0
+                    else "",
+                ),
+            )
+        except AssertionError:
+            traceback.print_exc()
```

Comment:
Not sure why this line.
Do you intend to log the error? If true, then why not `logger.exception` or `logger.error`?